certbot/src/certbot/crypto_util.py

```diff
@@ -397,6 +428,10 @@ def get_sans_from_cert(
     :rtype: list
 
     """
+    warnings.warn("get_sans_from_cert is deprecated and will be removed in the next "
+        "major release.", DeprecationWarning)
+    if typ is None:
+        typ = acme_crypto_util.Format.PEM
```

Comment:
another thing i think we could do if you wanted that strikes me personally as a bit nicer is to capture and ignore these warnings about acme_crypto_util.Format here and in the other functions below rather than propagating them to the caller

the fact that we still use acme_crypto_util.Format is an implementation detail the caller can't do anything about, we'll remove our usage when ripping out acme.crypto_util.format (as if we don't our tests of these functions will blow up), and it'd allow you to simplify capturing warnings in the unit tests of this file

like my other suggestion tho, if you don't think it's worth making this change, plz feel free to ignore this idea